### ISSUE TITLE
`ptxcompiler` Python 3.10 migration never sent

### ISSUE THREAD

**Issue by jakirkham** [regro/cf-scripts#1546]
The `ptxcompiler` feedstock appears not to have gotten a Python 3.10 migration PR. So submitted one manually ( https://github.com/conda-forge/ptxcompiler-feedstock/pull/13 ). It shows up in the "Done" list for the Python 3.10 migration. Though there does not appear to be a PR associated.

There have been other successful migrations of `ptxcompiler`. In the bot data for `ptxcompiiler`, do see [this entry where it appears the migration PR for Python 3.10 was closed]( https://github.com/regro/cf-graph-countyfair/blob/33ae1e4e3455c705dfd80c1101ee5a09c53a38dc/node_attrs/ptxcompiler.json#L3-L24 ), but maybe do some kind of unexpected event?

**Comment by jakirkham** [comment]
cc @bdice @gmarkall (for awareness)

**Comment by jakirkham** [comment]
Any thoughts on this one?

**Comment by beckermr** [comment]
I'd add the file manually and rerender to see what happens.

**Comment by jakirkham** [comment]
Yep that's what we did. More want to make sure that other bot PRs are not getting missing.

**Comment by minrk** [comment]
FWIW, this happened recently with openpmd-api being skipped and added to the done list for hdf51141 without ever getting a PR. I don't understand enough about the bot to try to dig into what incorrect signal is causing it to think the migration is done. A manual add and rerender went just fine: https://github.com/conda-forge/openpmd-api-feedstock/pull/107

**Comment by beckermr** [comment]
The bot will skip migrations where it adds the migrator and rerenders but nothing changes. The bot will also not send migrations where the solver checks do not pass. You can get info on that from the status page. 

IDK what happened here and so if none of these happened, then it is a real bug!
